android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/nuls/TestNULSSigner.kt

```diff
@@ -0,0 +1,40 @@
+package com.trustwallet.core.app.blockchains.nuls
+
+import com.google.protobuf.ByteString
+import com.trustwallet.core.app.utils.Numeric
+import com.trustwallet.core.app.utils.toHexByteArray
+import com.trustwallet.core.app.utils.toHexBytes
+import com.trustwallet.core.app.utils.toHexBytesInByteString
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+import wallet.core.jni.NULSSigner
+import wallet.core.jni.proto.NULS
+
+class TestNULSSigner {
+
+    init {
+        System.loadLibrary("TrustWalletCore")
+    }
+
+    @Test
+    fun NULSTransactionSigning() { 
+        val signingInput = NULS.SigningInput.newBuilder()
+            .setPrivateKey(ByteString.copyFrom(Numeric.hexStringToByteArray("9ce21dad67e0f0af2599b41b515a7f7018059418bab892a7b68f283d489abc4b")))
+            .setFrom("NULSd6Hgj7ZoVgsPN9ybB4C1N2TbvkgLc8Z9H")
+            .setTo("NULSd6Hgied7ym6qMEfVzZanMaa9qeqA6TZSe")
+            .setAmount(ByteString.copyFrom("0x989680".toHexByteArray()))
+            .setChainId(1)
+            .setIdassetsId(1)
+            .setNonce(ByteString.copyFrom("0000000000000000".toByteArray()))
+            .setRemark("")
+            .setBalance(ByteString.copyFrom("0x05f5e100".toHexByteArray()))
+            .setTimestamp(0x5d8885f8)
```

Comment:
seconds or milliseconds? can you add comments to proto file?

Reply:
second, I will add comment.